R/PipeOpFPCA.R

```diff
@@ -0,0 +1,95 @@
+#' @title Functional Principal Component Analysis
+#' @name mlr_pipeops_fda.fpca
+#'
+#' @format [`R6Class`] object inheriting from
+#' [`PipeOpTaskPreproc`][mlr3pipelines::PipeOpTaskPreproc]
+#'
+#' @description
+#' This is the class that extracts principal components from functional columns.
+#' See [`tfb_fpc()`][tf::tfb_fpc] for details.
+#'
+#' @section Parameters:
+#' The parameters are the parameters inherited from [`PipeOpTaskPreproc`], as well as the following parameters:
+#' * `pve` :: `numeric(1)` \cr
+#'   The percentage of variance explained that should be retained.
+#' * `n_components` :: `integer(1)` \cr
+#'   The number of principal components to extract.
+#'
+#' @section Naming:
+#' The new names generally append a `_pc_{number}` to the corresponding column name.
+#' If a column was called `"x"` and the there are three principcal components, the corresponding
+#' new columns will be called `"x_pc_1", "x_pc_2", "x_pc_3"`.
+#'
+#' @section Internals:
+#' Uses the [`tfb_fpc()`][tf::tfb_fpc] function.
+#'
+#' @section Methods:
```

Comment:
Not necessary if we use R6 roxygen documentation (I think this might be a copy-paste leftover from a time when I was using Martins custom documentation method)